ui/urls.py

```diff
@@ -26,8 +27,17 @@
     path('media/<pk>/edit', views.MediaEditView.as_view(), name='media_item_edit'),
     path('media/<pk>', views.MediaView.as_view(), name='media_item'),
     path('channels/<pk>', views.ChannelView.as_view(), name='channel'),
+    path(
+        'create_playlist',
```

Comment:
Could we have ``playlists/create`` instead? Or, if you're worried about future primary key collision, ``playlist/create``.